microShield.user.js

```diff
@@ -24,7 +24,7 @@
 // @match        *://sportalkorea.com/*
 // @match        *://*.sportalkorea.com/*
 //
-// @description        microShield allows the adblock community to resist against Ad-Shield widely.
+// @description  microShield allows the adblock community to resist against Ad-Shield widely.
```

Comment:
This can be used when providing multi-language description.
https://www.tampermonkey.net/documentation.php#_description